sdk/digitaltwins/azure-digitaltwins-core/pom.xml

```diff
@@ -43,6 +43,11 @@
       <artifactId>azure-core-http-netty</artifactId>
       <version>1.6.3</version> <!-- {x-version-update;com.azure:azure-core-http-netty;dependency} -->
     </dependency>
+    <dependency>
```

Comment:
Shouldn't there be a change to azure-core version in this file, too?

Reply:
azure core version was automatically updated when it was released.